### ISSUE TITLE
Possible issue with Deconvolution being synonym to Deisotoping (http://edamontology.org/operation_3629)

### ISSUE THREAD

**Issue by Kigaard** [edamontology/edamontology#746]
Currently "Deconvolution" is an exact synonym to Deisotoping (http://edamontology.org/operation_3629), which is highly related to proteomics/mass spectrometry. However, Deconvolution is also found in other fields than proteomics, such as genomics/sequencing.

**Comment by veitveit** [comment]
Makes sense. Can you give an example for usage of the term deconvolution in other fields?

The definition of the term is strongly related to spectra for now. Therefore, one would need to either generalize it to include other deconvolution types, or add the other ones to a different or new term.

**Comment by Kigaard** [comment]
@veitveit, one example in bio.tools is [DeCompress](https://bio.tools/decompress), which
> deconvolve targeted panels of mRNA expression into tissue compartment

Thereby possibly falling into the category of [Cellular deconvolution](https://en.wikipedia.org/wiki/Cellular_deconvolution).

The Deconvolution-definition is also found in the imaging-field, in regards to the removal of image blur.
